examples/tree-view/src/reducers/index.spec.js

```diff
@@ -1,163 +0,0 @@
-import deepFreeze from 'deep-freeze'
-import reducer from './index'
-import { increment, createNode, deleteNode, addChild, removeChild } from '../actions'
-
-describe('reducer', () => {
```

Comment:
:hand: We should be able to test the reducers in basically the same way we did before, since those are just `const actual = reducer(testState, action); expect(actual).toEqual(expected);`

In fact, if you switch the dispatched actions to the ones generated by `createSlice`, these reducer tests _should_ still pass.